libunwind/src/Registers.hpp

```diff
@@ -1924,6 +1973,8 @@ inline bool Registers_arm64::validRegister(int regNum) const {
 #ifdef __CHERI_PURE_CAPABILITY__
   if ((regNum >= UNW_ARM64_C0) && (regNum <= UNW_ARM64_C31))
     return true;
+  if (regNum == UNW_ARM64_ECSP)
```

Comment:
Do we need to permit clients to fetch the value of ECSP?  If not, we should remove `UNW_ARM64_ECSP` and support for it via the public register interface.

Reply:
Deleted this, we don't really want clients fetching this in any circumstance.